spec/yaml/swiftnav/sbp/specan.yaml

```diff
@@ -0,0 +1,65 @@
+# Packages define a logical collection of SBP messages. Packages can
```

Comment:
@mbavaro looks like you accidentally copied these comments over from the tutorial example....